core/src/main/scala/kafka/server/metadata/DynamicConfigPublisher.scala

```diff
@@ -103,9 +103,16 @@ class DynamicConfigPublisher(
               )
             case CLIENT_METRICS =>
               // Apply changes to client metrics subscription.
-              info(s"Updating client metrics subscription ${resource.name()} with new configuration : " +
-                toLoggableProps(resource, props).mkString(","))
-              dynamicConfigHandlers(ConfigType.ClientMetrics).processConfigChanges(resource.name(), props)
+              dynamicConfigHandlers.get(ConfigType.ClientMetrics).foreach(metricsConfigHandler =>
+                try {
+                  info(s"Updating client metrics ${resource.name()} with new configuration : " +
+                    toLoggableProps(resource, props).mkString(","))
```

Comment:
`toLoggableProps` by default treats the property as sensitive. Should we add the Client_Metrics type support in `KafkaConfig.loggableValue`?

Reply:
Done.